store/postgres/src/writable.rs

```diff
@@ -165,6 +170,7 @@ impl WritableStore {
             store.start_subgraph(logger, self.site.clone(), graft_base)?;
             self.store.primary_conn()?.copy_finished(self.site.as_ref())
         })
+        .await
```

Comment:
I did it because it was required at some point to have it `async`. So I would beed to bridge with some `async` and `non-async` at some point. What I mean is that if I revert `start_subgraph_deployment` asyncification, I would need to bridge `async` and `non-async` somewhere. If I keep it, I would need to do this 
`async`/`non-async` in another place.

What's the best way to keep `start_subgraph_deployment` async but fix the problem you mention here, without asyncify all `graft_pending` `start_subgraph` and `primary_conn`? Maybe the answer is to actually asyncify all those.

Reply:
The solution we had so far is to wrap it in `task::spawn_blocking` in the instance manager, we can continue to do that.

Reply:
I've reverted the deployment change and did use `futures03::executor::block_on` when I needed to bridge the inverse way.

Please review accordingly.